app/src/main/res/layout/activity_main.xml

```diff
@@ -6,248 +6,257 @@
-        android:text="@string/valueBaseText"
-        app:layout_constraintStart_toEndOf="@+id/lblGPSAcc"
-        app:layout_constraintTop_toTopOf="@+id/lblGPSAcc" />
-
-    <TextView
-        android:id="@+id/lblGPSAlt"
-        android:layout_width="100dp"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="25dp"
-        android:layout_marginTop="8dp"
-        android:text="Altitude:"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/lblGPSAcc" />
-
-    <TextView
-        android:id="@+id/vlGPSAlt"
-        android:layout_width="100dp"
-        android:layout_height="wrap_content"
-        android:text="@string/valueBaseText"
-        app:layout_constraintStart_toEndOf="@+id/lblGPSAcc"
-        app:layout_constraintTop_toTopOf="@+id/lblGPSAlt" />
-
-    <TextView
-        android:id="@+id/lblGPSSpeed"
-        android:layout_width="100dp"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="25dp"
-        android:layout_marginTop="8dp"
-        android:text="Speed:"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/lblGPSAlt" />
-
-    <TextView
-        android:id="@+id/vlGPSSpeed"
-        android:layout_width="100dp"
-        android:layout_height="wrap_content"
-        android:text="@string/valueBaseText"
-        app:layout_constraintStart_toEndOf="@+id/lblGPSAcc"
-        app:layout_constraintTop_toTopOf="@+id/lblGPSSpeed" />
-
-
-</androidx.constraintlayout.widget.ConstraintLayout>
+        <androidx.appcompat.widget.SwitchCompat
+            android:id="@+id/recorderSwitch"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:layout_marginTop="32dp"
+            android:layout_marginEnd="8dp"
+            android:checked="false"
+            android:text="Recording Off/On"
+            android:visibility="visible"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <androidx.appcompat.widget.SwitchCompat
+            android:id="@+id/swGPSRecord"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:checked="false"
+            android:text="Add GPS recording"
+            android:visibility="visible"
+            app:layout_constraintEnd_toEndOf="@+id/recorderSwitch"
+            app:layout_constraintTop_toBottomOf="@+id/recorderSwitch" />
+
+        <androidx.appcompat.widget.SwitchCompat
+            android:id="@+id/swGPSRecordingPrecision"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:checked="false"
+            android:text="GPS recording precision"
```

Comment:
Misschien voor duidelijker: text voor de switch 'low GPS precision' en text erna 'High GPS precision'

Reply:
Denk dat we in een apparte taak moeten kijken naar hoe we het beste de gebruiker moeten informeren